pastas/model.py

```diff
@@ -1314,7 +1325,7 @@ def _get_response(self, block_or_step, name, p=None, dt=None, add_0=False,
             timestep for the response function.
         add_0: bool, optional
             Add a zero at t=0.
-        kwargs
+        kwargs: dict
```

Comment:
mention to which method kwargs are passed

Reply:
done.